.github/workflows/cache.yml

```
@@ -0,0 +1,18 @@
+name: Cache .io Sites
+on: [deployment_status]
+jobs:
+  test:
+    if: github.event.deployment_status.state == 'success' && github.event.deployment.environment == 'production' && github.event.sender.id == 35613825
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version-file: '.nvmrc'
+      - name: Install npm@8
+        run: npm install --global npm@8
+      - name: Install dependencies
+        run: npm ci
+      - name: Cache .io sites
+        run: npx --package @hashicorp/platform-tools@0.5.0 hc-tools ./scripts/warm-dotio-cache.ts
```

nit: `@hashicorp/platform-tools` should be installed from the `npm ci` call above, so I believe we could do:
```suggestion
        run: npx hc-tools ./scripts/warm-dotio-cache.ts
```